### PR TITLE
WIP: Add release.yml to auto-release the latest binary

### LINKED ISSUE #3: Add Github action to auto-release the latest binary
To make it easy to get started with Fika, we need to automate the binary release
process. Check out https://github.com/spawnfest/bakeware and use github actions
to automate releases.

### PR DESCRIPTION
**Pull request by Atul9**
Closes #3

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      - name: Install zstd
+        run: sudo apt-get install zstd
+
+      - uses: actions/setup-elixir@v1
+        with:
+          otp-version: '22.2'
+          elixir-version: '1.10.4'
+
+      - name: Build
+        run: export MIX_ENV=prod && mix deps.get && mix release
```
Comment (polvalente) [inline .github/workflows/release.yml:39]: I think it would be better if `mix deps.get` was in a separate, cached step. My PR #18 adds this to `test.yml`

Comment (emilsoman) [reply .github/workflows/release.yml:39]: @polvalente  Is it possible to do `mix deps.get` once and use the cache in all workflows that need it?

Comment (polvalente) [reply .github/workflows/release.yml:39]: I don't really know. I'm more familiarized with CircleCI, so I'd have look into Github Actions a bit more. In CircleCI, we have the possibility of running steps one after the other, and some of them conditionally

Comment (emilsoman) [reply .github/workflows/release.yml:39]: @sreecodeslayer wanted to pitch in with his thoughts, but he's not able to comment here for some weird reason. @Atul9 please check with him on discord about his ideas

**Pull request closed without merging**